include/memkind.h

```diff
@@ -182,6 +182,13 @@ int memkind_create_kind(memkind_memtype_t memtype_flags,
 ///
 int memkind_destroy_kind(memkind_t kind);
 
+///
+/// \brief Get kind associated with allocated memory refernced by ptr
+/// \warning NON-STANDARD API
```

Comment:
What does it mean that the API is non-standard?
Remove this and add a note that this function has non-trivial performance overhead.